docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,104 @@
+
+There are many labels available, please feel free to put a label on your issue.
+
+## Requesting new features
+
+Stellio implements the NGSI-LD API specification.  You may find unsupported features.  
+When requesting a new feature, please provide the section and subsections related to the feature in the NGSI-LD 
+specification. Please provide if possible the version of the specification.
+
+Good example : 
+https://github.com/stellio-hub/stellio-context-broker/issues/12
+
+## Making Changes
+
+Contributions to Stellio are done using a pull request. Find below a description on how to do a good pull request :
+
+- Fork this repository
+- Create a branch on the forked repository.  
+A good branch name should follow this schema : `<label>/<issue_number>-<name>`
+
+    - `<label>` is the label of your issue : feature, bug, refactor, documentation...
+    - `<issue_number>` is the number of the issue
+    - `<name>` is the name of the feature, with `-` instead of spaces.
+
+For example : `documentation/60-add-contributing-file`
+
+- Commit your work and push to the branch.  
+A good commit message should follow this schema : `<label>: <commit msg> #<issue_number>`
+
+    - `<label>` is the label of your issue : feature, bug, refactor, documentation...
+    - `<commit_msg>` is the short description of your commit
+    - `<issue_number>` is the number of the issue
+
+For example : `documentation: add contributing.md file #60`
+
+- Submit a pull request to the target branch in the main repository
+- Wait for reviews from [Stellio maintainers](https://github.com/orgs/stellio-hub/people)
+- Stay reactive to comments
+
+When a user is starting a comment thread, he should resolve it once he thinks the discussion has ended.
+
+Once there is agreement that the code is ready to be included in the main project, one of Stellio's
+maintainers will merge your contribution.
+
+Increase the chances that your pull request will be accepted and gain time :
+
+- Follow our [coding style](https://github.com/stellio-hub/stellio-context-broker/blob/develop/docs/CONTRIBUTING.md#coding-style).
+- Write tests for your changes.
+- Write a good commit title, add description if there are many changes.
+
+If your pull request is too big, you might want to split it into smaller pull requests.
+
+Small pull requests lead to fast reviews and a better understanding of the changes.
+
+A good guide on why you should split a pull request and how : https://www.thedroidsonroids.com/blog/splitting-pull-request
+
+## Project Structure
+
+Stellio is divided into five main modules: api-gateway, entity-service, search-service, subscription-service and shared.
+Each module follows the same schema :
+- **src**:
+    - **main**: contains the source code.
+        - **kotlin**: contains the source kotlin code.
+        - **resources** : contains the resources.
+    - **test**: contains the test code, a unit test class of a source code class should have the same package.
```

Comment:
Maybe you can mention that the test folder contains also two sub-folders Kotlin and resources.
And the resources contains the test data.

Reply:
Yep Indeed, the api-gateway module is not containing any resources folder but this is not a matter I think I can add the sub-folders here..